azure/packages/azure-client/api-report/azure-client.api.md

```diff
@@ -40,6 +40,7 @@ export class AzureClient {
 export interface AzureClientProps {
     readonly configProvider?: IConfigProviderBase;
     readonly connection: AzureRemoteConnectionConfig | AzureLocalConnectionConfig;
+    readonly isEphemeralContainer?: boolean;
```

Comment:
1 `AzureClient` instance can be used to create many containers, so I think `useEphemeralContainers` would be better. I made this comment in #16006 but it was lost in the conversation. @pradeepvairamani any insights?

Reply:
I don't have a strong opinion on this. Since this boolean is representing if the container that is created by this AzureClient is ephemeral or not, it would be better to use isEphemeralContainer. useEphemeralContainers sounded to me like it is something outside of the artifact that is being created by AzureClient. But like I said, I am totally fine with either one of the names.

Reply:
Makes sense :)